ldk/javascript/src/types/oliveHelps/whisper.d.ts

```diff
@@ -350,6 +351,23 @@ declare namespace WhisperService {
     onChange: WhisperHandlerWithParam<boolean>;
   };
 
+  type Rating = Component<'rating'> & {
+    defaultValue?: number;
+    disabled?: boolean;
+    emptyIcon?: string;
+    emptyIconColor?: string;
+    emptyIconType?: string;
+    icon?: string;
+    iconColor?: string;
+    max?: number;
+    name?: string;
+    onChange?: WhisperHandlerWithParam<number>;
+    precision?: number;
+    readOnly?: boolean;
+    size?: string;
+    value?: number;
+  };
```

Comment:
If all of these are optional, what happens if you use this component without any props?

Reply:
There are default values configured on the backend. It renders as a component with stars, 3 / 5 with the 3 being filled.